cumulus_library/template_sql/core_medication.sql.jinja

```diff
@@ -0,0 +1,105 @@
+CREATE TABLE core__medications AS (
```

Comment:
MINOR: Should be "core_medication" (drop the s)

Reply:
can do